spec/menu_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "menu"
+
+describe Menu do
+  context "#see" do
+    let(:menu) { described_class.new }
+    it "prints out menu" do
+      expect { menu.see }.to output.to_stdout
+    end
+  end
+
+  context "#choose" do
+    let(:order_double) { double :Order_Double, add: "Dish added to order" }
+    let(:menu) { described_class.new(order_double) }
+    it "adds dish to order" do
+      expect(menu.choose("Chocolate Frogs", 10)).to eq("Dish added to order")
+    end
+  end
+
+  context "#confirm_order" do
+    let(:order_double) { double :Order_Double, confirm: true }
+    let(:menu) { described_class.new(order_double) }
```

Comment:
Good use of doubles, but it would be better to isolate all the tests by setting up the doubles on line 5, rather than within individual context blocks.